hack/test-cmd.sh

```diff
@@ -337,12 +330,19 @@ for test in "${tests[@]}"; do
   os::cmd::try_until_text "oc get projects -o name" "project/${namespace}"
   os::test::junit::declare_suite_end
 
-  ${test}
+  if ! ${test}; then
```

Comment:
The placement inside of the `if` *will* fail to send `ERR` if the test fails, so `set -o errexit` and our `ERR` trap (the stacktrace) won't fire, but that's on purpose.